### PR TITLE
Manage scheduled reports

### LINKED ISSUE #23: Scheduled reports list
It would be great to have a CLI command to list all the scheduled tests, in order to review them before decide to schedule a new report.

### PR DESCRIPTION
**Pull request by fquffio**
This PR resolves #23 and introduces both an endpoint to list reports scheduled to be generated in the future, and a CLI to manage upcoming scheduled reports.

a11ygator-cli-manage

### CODE REVIEW

**Comment by nicolocarpignoli** [comment]
Great! Is this still in WIP because we want also to add deletion of scheduled reports? Maybe I will stop review waiting for pending changes

**Comment by fquffio** [comment]
@nicolocarpignoli I’ve removed the “WIP”. It should be complete now. See the attached GIF for usage. If it looks alright, we should update the docs (`docs/cli.md`) too.

**Comment by fquffio** [comment]
@nicolocarpignoli Docs updated. 👍

File: lambda/report-delete/index.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    S3.deleteObject({å
+      Bucket: S3_BUCKET,
+      Key: fullPageScreenshotKey,
+    }).promise(),
+  ]);
+  console.timeEnd('Deleting report data from S3');
+};
+
+/**
+ * Stop step function execute that would eventually enqueue report generation.
```
Comment (nicolocarpignoli) [inline lambda/report-delete/index.js]: I guess here using 'execution' is more correct than 'execute'

**Pull request merged**